10331.xml

```diff
@@ -0,0 +1,221 @@
+    <ObjectURN>urn:oma:lwm2m:x:10331</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Single</MultipleInstances>
+    <Mandatory>Mandatory</Mandatory>
+    <Resources>
+      <Item ID="1">
+        <Name>Current status</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..127</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Current status, for example: working, pasued, dormant, holding, charging and so on.]]></Description>
+      </Item>
+      <Item ID="2">
+        <Name>Services Providing</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..127</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Providing services: for example: patrolling, advertisement playing, patrolling and so on,
+			one robot can do several works at the same time, for example, patrolling with advertisement playing.]]></Description>
+      </Item>
+	  <Item ID="3">
+		<Name>Advertising Contents</Name>
+		<Operations>R</Operations>
+		<MultipleInstances>Single</MultipleInstances>
+		<Mandatory>Mandatory</Mandatory>
+		<Type>String</Type>
+		<RangeEnumeration/>
+		<Units/>
+		<Description><![CDATA[The Advertising Contents it is playing.]]></Description>
+	  </Item>
+      <Item ID="4">
+        <Name>Current Language</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..127</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Current language used by the robot.]]></Description>
+      </Item>
+      <Item ID="5">
+        <Name>Volume</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[The Volume of the speaker.]]></Description>
+      </Item>	  
+      <Item ID="6">
+        <Name>Moving Status</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..2</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[0:Moving, 1:Stopped, 2:Suspended.]]></Description>
+      </Item>
+      <Item ID="7">
+        <Name>Moving Speed</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>M/H</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
M/H is the unit of speed, how much meters per hour.